CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 
 ## Added
 
+- [#3317](https://github.com/wasmerio/wasmer/pull/3317) Add a `wasmer add` command for adding bindings for a WAPM package to your project (only Python and JavaScript are supported for now)
```

Comment:
By the way, you don't need to update the changelog at all in PRs. When doing a release, I just do `gh search --prs --merged --repo wasmerio/wasmer` and then copy the list since the last release. Otherwise I'd have to look if every PR has included this message. So this commit is nice, but pointless.

Reply:
Ah okay. We should probably update the pull request template so we don't tell people to update the changelog.